razorpay-subscriptions.php

```diff
@@ -341,3 +347,13 @@ function razorpay_webhook_subscription_init()
 
     $rzpWebhook->process();
 }
+
+function subscriptionInfoLog($message){
```

Comment:
@sonyy-m Move this to a separate file refer this one https://github.com/razorpay/razorpay-woocommerce/blob/master/includes/debug.php